### PR TITLE
feat(grafana): add cas scheduler dashboards

### PR DESCRIPTION
**Pull request by v-stickykeys**
- Adds dashboards to show health, cpu, and memory of cas scheduler
- Updates CAS API dashboards to only show internet facing load balancer used for API
-  Updates alert message to be more descriptive
- Adds count for current number of CAS workers active

Screen Shot 2022-10-18 at 5 28 46 PM
Screen Shot 2022-10-18 at 5 29 04 PM

### Notes

If approved for dev I'll update this PR with the same for other envs

### CODE REVIEW

File: grafana/provisioning/dashboards/dev/anchorservice.json
Comment (gvelez17) [inline grafana/provisioning/dashboards/dev/anchorservice.json:248]: are we intending to replace this?

**Review by gvelez17 — APPROVED** [review]

File: grafana/provisioning/dashboards/dev/anchorservice.json
Comment (gvelez17) [inline grafana/provisioning/dashboards/dev/anchorservice.json:91]: is ECS/ContainerInsights available on clay and main?  When I tried it i only saw it on dev

File: grafana/provisioning/dashboards/dev/anchorservice.json
Comment (gvelez17) [inline grafana/provisioning/dashboards/dev/anchorservice.json:95]: this does seem like the same number being counted in pr 48 at https://github.com/3box/ceramic-stats/pull/48/files#diff-c5c6048e4df66fd4bf2d0c16357ff399cbe9b268aa2e6c5bf12d90a15285e98bR196

File: grafana/provisioning/dashboards/dev/anchorservice.json
Comment (gvelez17) [reply grafana/provisioning/dashboards/dev/anchorservice.json:91]: I did use it on dev but had to use just AWS/ECS on clay and main

Comment (v-stickykeys) [reply grafana/provisioning/dashboards/dev/anchorservice.json:91]: yeah I will wait to merge this until enabled on the other environments too

File: grafana/provisioning/dashboards/dev/anchorservice.json
Comment (v-stickykeys) [reply grafana/provisioning/dashboards/dev/anchorservice.json:95]: yes that's correct. it is the same as what you have. I'll merge this after you do yours and resolve the conflicts

File: grafana/provisioning/dashboards/dev/anchorservice.json
Comment (v-stickykeys) [reply grafana/provisioning/dashboards/dev/anchorservice.json:248]: It is just moved above on line 878

**Comment by v-stickykeys** [comment]
fyi @gvelez17 we have been following the norm that the person who creates the pr merges. I was planning to add more to this before merging in but I should have kept it in draft so it's my bad

**Pull request merged**
